src/PaymentGateways/Gateways/PayPal/PayPalStandardGateway/PayPalStandardGateway.php

```diff
@@ -9,12 +9,21 @@
 use Give\Framework\PaymentGateways\Traits\HandleHttpResponses;
 use Give\Framework\Support\Scripts\Concerns\HasScriptAssetFile;
 use Give\PaymentGateways\Gateways\PayPalStandard\PayPalStandard;
+use Give\PaymentGateways\Gateways\PayPalStandard\Views\PayPalStandardBillingFields;
 
 class PayPalStandardGateway extends PayPalStandard
 {
     use HandleHttpResponses;
     use HasScriptAssetFile;
 
+    /**
+     * @inheritDoc
+     */
+    public function getLegacyFormFieldMarkup(int $formId, array $args): string
+    {
+        return (new PayPalStandardBillingFields())($formId);
+    }
+
```

Comment:
Does this change have any relationship with the currency switcher?

Reply:
nope! moved this to develop efaad7a0